flexdll.c

```diff
@@ -47,8 +47,8 @@ typedef struct dlunit {
 } dlunit;
 typedef void *resolver(void*, const char*);
 
-static int error = 0;
-static char error_buffer[256];
+static __thread int error = 0;
+static __thread char error_buffer[256];
```

Comment:
I think this should work:
```suggestion
#ifdef _MSC_VER
#define __thread __declspec(thread)
#endif
static __thread int error = 0;
static __thread char error_buffer[256];
```